### PR TITLE
uWSGI tuning

### PR DESCRIPTION
**Pull request by robhudson**

### CODE REVIEW

**Review by pmac — APPROVED** [review]
🎂

File: bin/uwsgi.ini [hunk 1]
```diff
@@ -1,8 +1,32 @@
...
-enable-threads = true
-lazy-apps = true
+strict = true  ; Fail on unknown config options
+
+http = :8000
+module = basket.wsgi:application
+
+die-on-term = true  ; Shutdown on SIGTERM (default is respawn)
+enable-threads = true  ; Support multithreading
+harakiri = 20  ; Kill hung requests after 20 seconds
```
Comment (stevejalim) [inline bin/uwsgi.ini:10]: I think gunicorn's default is 30secs. I doubt we'll notice but worth keeping an eye on if we get more timeouts

File: bin/uwsgi.ini [hunk 2]
```diff
@@ -1,8 +1,32 @@
...
+lazy-apps = true  ; Load apps in each worker instead of the master
 max-requests = 5000
+need-app = true  ; Fail if app can't be loaded
+single-interpreter = true
+vacuum = true  ; Delete socket files on exit
+
+# The ELB has request logs, so we don't need them here
```
Comment (stevejalim) [inline bin/uwsgi.ini:17]: SMART!

**Review by stevejalim — APPROVED** [review]
Lots to like there. Thanks! r+++

**Pull request merged**